### ISSUE TITLE
Shouldn't allow the space between `/` and the service name while writing a service in Ballerina

### ISSUE THREAD

**Issue by gabilang** [ballerina-platform/ballerina-lang#29680]
**Description:**
VSCode plugin didn't detect when a space was given between `/` and the service name as follows
`service / hello on serverEP`

![image](https://user-images.githubusercontent.com/35595065/112141633-8804cc00-8bfb-11eb-9dd4-dc12f9850957.png)

**Suggested Labels:**
Team/LanguageServer, Version/SLAlpha3

**Suggested Assignees:**

**Affected Product Version:**
SLAplha3

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by hasithaa** [comment]
@rdulmina @lochana-chathura Looks like a bug parsing the identifier.

**Comment by lochana-chathura** [comment]
Raised this in the spec: ballerina-platform/ballerina-spec#792

**Comment by lochana-chathura** [comment]
@hasithaa space is allowed according to https://github.com/ballerina-platform/ballerina-spec/issues/792#issuecomment-809300336

**Comment by hasithaa** [comment]
Ack. We can improve the formatter. @IrushiL Please note.

**Comment by hasithaa** [comment]
A second thought, I feel the current formatter output is correct in this case. what other things?

**Comment by lochana-chathura** [comment]
> A second thought, I feel the current formatter output is correct in this case. What do others think?

yeah. Formatter's behaviour is correct. I think we need to support syntax highlighting for both scenarios.
